packages/kbn-monaco/src/register_globals.ts

```diff
@@ -6,6 +6,7 @@
  * Side Public License, v 1.
  */
 
+import 'remote-web-worker';
```

Comment:
@jbudz should we consider moving this to `packages/kbn-ui-shared-deps-src/src/polyfills.js` or somewhere more shared since it is patching the global worker object?